sci-mathematics/octave/octave-9.2.0.ebuild

```diff
@@ -110,36 +97,13 @@ BDEPEND="
 		dev-texlive/texlive-metapost
 		virtual/latex-base
 	)
-	qt5? ( dev-qt/linguist-tools:5 )
```

Comment:
This one we should probably replace with `gui? ( dev-qt/qttools:6[linguist )` (unless of course qt linguist is no longer required in the qt6 version)

Reply:
it compiles without that

Reply:
> uhm, actually I have already installed `dev-qt/qttools:6[linguist]` on my system so I am not sure

Lets just keep it in then, just to be sure. I can test later if it is actually used.